code/modules/antagonists/heretic/items/madness_mask.dm

```diff
@@ -0,0 +1,74 @@
+// The spooky "void" / "abyssal" / "madness" mask for heretics.
+/obj/item/clothing/mask/madness_mask
```

Comment:
consider making this a focus but nerfing it's abilities more? there is always a room for a nerf to abyssal mask